packages/arb-token-bridge-ui/src/pages/index.tsx

```diff
@@ -103,6 +121,12 @@ export function getServerSideProps({
     }
   }
 
+  if (
+    process.env.NODE_ENV !== 'production' ||
+    process.env.NEXT_PUBLIC_IS_E2E_TEST
+  ) {
+    await registerLocalNetwork()
```

Comment:
We need to register networks so that it recognises L3 local and doesn't change the change to default